pkg/clusteragent/admission/start.go

```diff
@@ -35,6 +36,7 @@ type ControllerContext struct {
 	Client              kubernetes.Interface
 	StopCh              chan struct{}
 	ValidatingStopCh    chan struct{}
+	Demultiplexer       demultiplexer.Component
```

Comment:
nit [consistency]: why did you choose to pass the demultiplexer component through the `ControllerContext` struct? All of the other components are being passed through as parameters.

Reply:
Resolved together.